onboarding-checklist.md

```diff
@@ -54,11 +54,14 @@ make sure you're assigned to this issue.
 
 #### Mandatory training ####
 
-- [ ] Complete [PALMS Cybersecurity Training](https://etms.hq.dhs.gov),
+- [ ] Complete ~~[PALMS Cybersecurity Training](https://etms.hq.dhs.gov)~~,
 including Mandatory Cyber Security and Privacy Training, and accepting
 the Rules of Behavior, which is required before we can give you access to any
 systems, like Trio.
 
+- PALMS has been retired, and is currently being replaced by the
```

Comment:
```suggestion
- PALMS has been retired, and is in the process of being replaced by the
```

Reply:
@jsf9k The current site is an interim replacement until a new final solution is in place.  I can reword it, but don't think your recommendation is accurate

Reply:
I see.  Maybe change "currently" to "temporarily" then?

Reply:
Fixed in commit d434617.